README.md

```diff
@@ -1,46 +1,44 @@
+
 # CLMM
 [![Build Status](https://travis-ci.org/LSSTDESC/CLMM.svg?branch=master)](https://travis-ci.org/LSSTDESC/CLMM) 
 [![Build and Check](https://github.com/LSSTDESC/CLMM/workflows/Build%20and%20Check/badge.svg)](https://github.com/LSSTDESC/CLMM/actions?query=workflow%3A%22Build+and+Check%22)
 [![Coverage Status](https://coveralls.io/repos/github/LSSTDESC/CLMM/badge.svg?branch=master)](https://coveralls.io/github/LSSTDESC/CLMM?branch=master)
 
-The LSST-DESC Cluster Lensing Mass Modeling (CLMM) code is a Python library for performing galaxy cluster weak lensing analyses.
-clmm is associated with Key Tasks _DC1 SW+RQ_ and _DC2 SW_ of the LSST-DESC [Science Roadmap](https://lsstdesc.org/sites/default/files/DESC_SRM_V1_4.pdf) pertaining to absolute and relative mass calibration.
-CLMM is descended from [clmassmod](https://github.com/deapplegate/clmassmod) but distinguished by its modular structure and scope, which encompasses both simulated data sets with a known truth and observed data from which we aim to discover the truth.
+The LSST-DESC Cluster Lensing Mass Modeling (CLMM) code is a Python library for performing galaxy cluster mass reconstruction from weak lensing observables. CLMM is associated with Key Tasks _DC1 SW+RQ_ and _DC2 SW_ of the LSST-DESC [Science Roadmap](https://lsstdesc.org/sites/default/files/DESC_SRM_V1_4.pdf) pertaining to absolute and relative mass calibration.
+<!---CLMM is descended from [clmassmod](https://github.com/deapplegate/clmassmod) but distinguished by its modular structure and scope, which encompasses both simulated data sets with a known truth and observed data from which we aim to discover the truth.--->
 The documentation of the code can be found [here](http://lsstdesc.org/CLMM/).
 
 ## Table of contents
 1. [Requirements](#requirements)
 2. [Installation](#installation)
 3. [Overview](#overview)
-3. [Contributing](#contributing)
+4. [Contributing](#contributing)
+5. [Contact](#contact)
 
 ## Requirements <a name="requirements"></a>
 
-CLMM requires Python version 3.6 or later.  To run the code, there are the following dependencies:
+CLMM requires Python version 3.6 or later.  CLMM has the following dependencies:
 
 - [numpy](http://www.numpy.org/) (1.17 or later)
```

Comment:
Referring as NumPy if we want to be really accurate.

Reply:
Done.